Organization-for-CS-Build-Week-1/CS-Build-Week-1-Flask
# Makes Walker randomness dynamic, saves map seed

# Changelog

### Map
- changes number of Walkers to 2
- adds `rand_factor` parameter to Walker class
- generates seed for `random`
- returns seed from `generate_grid()`

### Worlds Model
- adds `map_seed` column

### Blueprints
- adds `worlds_blueprint`

### App
- passes `map_seed` to `Worlds` as necessary

#### To test:
1. Start server locally.
2. Send GET request to `baseURL/api/check`.
3. Send GET request to `baseURL/api/models/world` with `request.headers = {Admin:True}`.
4. You should see an array of one object including a `map_seed` property with an `int` value between 0 and 10^6.
